.claude/skills/recommend-commit-message/SKILL.md

```
@@ -0,0 +1,160 @@
+
+- 기능 추가 + 버그 수정이 함께 있으면 → 분리 제안 (각각 `feat`, `fix`)
+- 기존 코드를 새 패턴으로 교체했으나 동작이 같으면 → `refactor`
+- 파일 이동만 있으면 → `move`
+- Gradle 버전/의존성만 변경되면 → `build`
+- GitHub Actions, CI 설정만 변경되면 → `ci`
+
+---
+
+## 출력 형식
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)